vdev/src/testing/integration.rs

```diff
@@ -277,7 +277,7 @@ impl Compose {
 
     fn run(&self, action: &str, args: &[&'static str], config: Option<&Environment>) -> Result<()> {
         let mut command = CONTAINER_TOOL.clone();
-        command.push("-compose");
+        command.push(" compose");
```

Comment:
I think this should be an argument passed to the `docker` command.

Reply:
Doh, yes thank you!

Reply:
98a97ee61c